uamqp/client.py

```diff
@@ -871,7 +874,8 @@ def __init__(
         self._last_activity_timestamp = None
         self._was_message_received = False
         self._message_received_callback = None
-        self._received_messages = None
+        self._streaming_receive = False
+        self._received_messages = compat.queue.Queue()
```

Comment:
These two lines don't seem to be in `client_async.close_async()`?

Reply:
you mean resetting these two variables when a client is closed? but we don't support open-close-open..